test/react_test/__snapshots__/Link.react.test.js.snap

```diff
@@ -0,0 +1,34 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
@tegsheeb Is this example data? Or are these results from your tests?

Reply:
This is the test result from sample test Link.react.test.js. In jest, react-test-renderer takes the snapshot (like in this example) in the end-to-end test and output as a result. When any changes are made to the code being test, jest compares it with the previous snapshot.